index.html

```diff
@@ -2370,7 +2370,7 @@ <h2 id="utility">Utility Functions</h2>
         <b class="header">escape</b><code>_.escape(string)</code>
         <br />
         Escapes a string for insertion into HTML, replacing
-        <tt>&amp;</tt>, <tt>&lt;</tt>, <tt>&gt;</tt>, <tt>&quot;</tt>, <tt>&#96;</tt>, and <tt>&#x27;</tt> characters.
+        <tt>&amp;</tt>, <tt>&lt;</tt>, <tt>&gt;</tt>, <tt>&quot;</tt>, <tt>&#x60:</tt>, and <tt>&#x27;</tt> characters.
```

Comment:
You accidentally typed a colon instead of a semicolon. Please push another commit to fix this.

```suggestion
        <tt>&amp;</tt>, <tt>&lt;</tt>, <tt>&gt;</tt>, <tt>&quot;</tt>, <tt>&#x60;</tt>, and <tt>&#x27;</tt> characters.
```

Reply:
My bad, I have corrected it along with documentation of `_.unescape `